README.md

```diff
@@ -1,10 +1,164 @@
-# Python Starter
+# Assembly interfaces analysis 
 
-Starter repo for Python Projects. Should be used to get common linting and formatting started
+## Basic information
 
+This python package works with PISA-Lite to analyse data for macromolecular interfaces and interactions in assemblies.
 
-pre-commit usage
------------------
+The code will:
+- Analyse macromolecular interfaces with PISA
+- Create Json dictionary with assembly interactions/interfaces information
+
+```
+git clone https://github.com/PDBe-KB/pisa-analysis
+
+cd pisa-analysis 
+``` 
+## Dependencies 
+
+The process runs PISA-Lite as a subprocess and requires apriori compilation of PISA. For more information on how to compile PISA-LITE visit our internal page: 
+
+[PISA-Lite documentation](https://www.ebi.ac.uk/seqdb/confluence/pages/viewpage.action?spaceKey=PDBE&title=Interaction+and+interfaces+-+assemblies)
```

Comment:
@Grisell Diaz Leines: This is not a public document. So we can't give reference to this.